cmd/juju/status/formatted.go

```diff
@@ -33,6 +33,7 @@ type errorStatus struct {
 
 type modelStatus struct {
 	Name             string             `json:"name" yaml:"name"`
+	Type             string             `json:"type,omitempty" yaml:"type,omitempty"`
```

Comment:
shouldn't a model always have a type?

Reply:
Not for older servers which don't provide one. What I should do is use "iaas" if it comes across the wire without type set.